struk-semantik.tex

```diff
@@ -172,32 +172,32 @@ \subsection{Eigenschaften der SOS}
     \emph{Induktionsschritt:} $k - 1 \mapsto k$ mit $k \geq 3$
 
     Betrachten den ersten Schritt $\langle S_1; S_2, \sigma \rangle \Rightarrow^k \sigma''$.
-    Zwei Fälle
+    Zwei Fälle:
     \begin{enumerate}
-        \item $\infruleSos[1]{seq}$ Der erste Schritt hat die Form $\langle S_1; S_2, \sigma \rangle \Rightarrow^k \langle S_1'; S_2, \sigma''' \rangle$
+        \item $\infruleSos[1]{seq}$ Der erste Schritt hat die Form $\langle S_1; S_2, \sigma \rangle \Rightarrow \langle S_1'; S_2, \sigma''' \rangle$
```

Comment:
Ich glaube, das war in meiner Version korrekt. Passt auch zum Buch:
<img width="756" alt="Screenshot 2022-07-27 at 11 02 09" src="https://user-images.githubusercontent.com/6906734/181207769-836c9cd3-550d-4155-8803-b066bcde67f9.png">

Reply:
Siehe 4ca8ada0d40540b1b69e0b46a3f57a6d50e28939

Reply:
Hm, da war ich mir auch sehr unsicher. Ich dachte halt, der *erste Schritt* müsse Länge 1 haben, also nicht k. Sonst wäre die Summe der Schritte am Ende ja auch k+k-1=2k-1 anstatt k.
![image](https://user-images.githubusercontent.com/62887033/181214265-f4437471-f0a3-4940-9cc9-57aaec3a492a.png)
Das ist glaub ich die relevante Stelle im Buch (Seite 37 und 38)

Reply:
@jneuendorf Kannst du dir das nochmal anschauen? Bzw. mich aufklären, ob ichs falsch verstanden hab? :)

Reply:
Korrekt ist die Version $\Rightarrow$ und **nicht** $\Rightarrow^k$. Zum einen heißt es ja explizit der **erste Schritt**.
Passt sowohl zum Bild von @akriese als auch zu dem Teil ganz unten im Bild von @jneuendorf.
Es geht ja hier darum, dass man zwischen den beiden Möglichkeiten für die Sequenz unterscheidet. In ersten Fall benötigen wir mehr als einen Schritt, um $S_1$ abzuarbeiten, im zweiten Fall benötigt $S_1$ genau einen Schritt.